<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>650</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NACA - Biagio Raucci</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>19</x>
     <y>19</y>
     <width>611</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="qvb_grafico"/>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>NACA</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QCheckBox" name="cB_griglia">
        <property name="text">
         <string>griglia</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lE_formula"/>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>grafico</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lE_formula</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>cB_griglia</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cB_griglia</sender>
   <signal>stateChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>cB_grigliaChange()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>574</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>634</x>
     <y>366</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>pB_graficoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>587</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lE_formula</sender>
   <signal>returnPressed()</signal>
   <receiver>Form</receiver>
   <slot>pB_graficoClick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>362</y>
    </hint>
    <hint type="destinationlabel">
     <x>648</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>cB_grigliaChange()</slot>
  <slot>pB_graficoClick()</slot>
 </slots>
</ui>
